Changelog.md

```diff
@@ -9,6 +9,8 @@ Compiler Features:
  * Yul Optimizer: Add rule to convert `mod(mul(X, Y), A)` into `mulmod(X, Y, A)`, if `A` is a power of two.
  * Yul Optimizer: Add rule to convert `mod(add(X, Y), A)` into `addmod(X, Y, A)`, if `A` is a power of two.
  * Code Generator: More efficient code for checked addition and subtraction.
+ * Commandline Interface: Add `--no-cbor-metadata` that skips metadata from getting appended at the end of the deployed bytecode.
+ * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips metadata from getting appended at the end of the deployed bytecode.
```

Comment:
```suggestion
 * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the deployed bytecode.
 * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the deployed bytecode.
```

Should I use this? Feels off.

Reply:
Why? Seems fine for me. @ekpyron what do you think?

Reply:
Doesn't seem too off to me. ``CBOR-encoded`` would also work.

Reply:
Yeah, but the point was that we agreed to call it "CBOR metadata". I.e. it's not just any piece of CBOR-encoded medata. It's a very specific one :)

Reply:
Sure. Fixing now.